jwcrypto/jwk.py

```diff
@@ -688,7 +710,7 @@ def _get_public_key(self, arg=None):
         elif self._params['kty'] == 'EC':
             return self._ec_pub(self._key, arg).public_key(default_backend())
         elif self._params['kty'] == 'OKP':
-            return self._okp_pub(self._key, arg).public_key(default_backend())
+            return self._okp_pri(self._key).public_key()
```

Comment:
This will break if you have only a public key in the JWK I think.

Reply:
Yes. Will fix.